runtime/ftplugin/objdump.vim

```diff
@@ -0,0 +1,18 @@
+" Vim filetype plugin file
+" Language:     Objdump
+" Maintainer:   Colin Kennedy <colinvfx@gmail.com>
+" Last Change:  2023 October 25
+
+if exists("b:did_ftplugin")
+  finish
+endif
+
+let s:cpo_save = &cpo
+set cpo&vim
+
+let b:did_ftplugin = 1
+
+setlocal commentstring=#\ %s
```

Comment:
This option needs to be added to `b:undo_ftplugin`.

```vim
let b:undo_ftplugin = "setlocal cms<"
```

Reply:
Done!